neuron_explainer/activations/derived_scalars/README.md

```diff
@@ -1,63 +1,149 @@
+> The most bare-bones function that shows the steps outlined above is in [activation_server/derived_scalar_computation:get_derived_scalars_for_prompt](../../../neuron_explainer/activation_server/derived_scalar_computation.py)
+
+---
+
+## How to define a new `ScalarDeriver`
+
+When defining a new `ScalarDeriver` that does not correspond to an `ActivationLocationType`, the user provides:
+
+1. The information necessary to compute the derived scalar. 
+
+    This includes:
+
+   1. The `ScalarSources` it expects
 
+   2. The `tensor_calculate_derived_scalar_fn`, which takes tensors as arguments corresponding to the `ScalarSources` as well as the `layer_index` and `pass_type`. 
 
-When defining a new `ScalarDeriver` that does not correspond to an `ActivationLocationType`, the user provides (1) the information necessary to compute the derived scalar. This includes (a) the `ScalarSources` it expects, (b) `tensor_calculate_derived_scalar_fn`, which takes as arguments tensors corresponding to the `ScalarSources` as well as the `layer_index` and `pass_type`. This lives in a function called `make_..._scalar_deriver`. These can be specified implicitly, if the new scalar can be derived from a transformation on one or more pre-existing derived scalars. The `make_..._scalar_deriver` functions are associated to DSTs in `derived_scalars/make_scalar_derivers.py`. (2) a specification of the shape of the output, in terms of `Dimension` objects. This lives in `derived_scalars/derived_scalar_types.py`.
+        - This lives in a function called `make_..._scalar_deriver`. These can be specified implicitly, if the new scalar can be derived from a transformation on one or more pre-existing derived scalars. 
 
-`ScalarDeriver` is meant to be the primary class used to refer to activations once they are READ from disk. When WRITTEN to disk, the primary class used is `ActivationLocationType`, since we always want to save the least processed form of the data possible.
+        - The `make_..._scalar_deriver` functions are associated to DSTs in [derived_scalars/make_scalar_derivers.py](../../../neuron_explainer/activations/derived_scalars/make_scalar_derivers.py).
 
+2. A specification of the shape of the output, in terms of `Dimension` objects. 
 
-TO DEFINE A NEW DERIVED SCALAR TYPE (DST):
+   - This lives in [derived_scalars/derived_scalar_types.py](../../../neuron_explainer/activations/derived_scalars/derived_scalar_types.py)
 
-1. Add a new `DerivedScalarType` to the Enum in `scalar_deriver.py`.
+> `ScalarDeriver` is meant to be the primary class used to refer to activations once they are READ from disk. 
+>
+> When WRITTEN to disk, the primary class used is `ActivationLocationType`, since we always want to save the least processed form of the data possible.
+
+---
+
+## How to define a new `DerivedScalarType`
+
+1. Add a new `DerivedScalarType` to the Enum in [scalar_deriver.py](../../../neuron_explainer/activations/derived_scalars/scalar_deriver.py):
```

Comment:
Replace the colon at the end with a period.

Reply:
And fixed this - [Fix derived_scalars readme](https://github.com/openai/transformer-debugger/pull/11/commits/c91b0fd47d813e328832f036f49544db06eb7fac)